main.go

```diff
@@ -0,0 +1,119 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"strconv"
+	"time"
+)
+
+func main() {
+	// Exit after 2hours 30 mins
+	time.AfterFunc(150*time.Minute, func() { failf("Session Timed Out") })
+
+	log.Print("Starting the build on BrowserStack App Automate")
+
+	username := os.Getenv("browserstack_username")
+	access_key := os.Getenv("browserstack_accesskey")
```

Comment:
Should we move these into contant file and consume the constant here i.e BROWSERSTACK_USERNAME := os.Getenv("browserstack_username")